src/UserConsentListBuilder.php

```diff
@@ -65,10 +66,11 @@ public function buildHeader() {
    * {@inheritdoc}
    */
   public function buildRow(EntityInterface $entity) {
+    $owner = $entity->getOwner();
     /* @var $entity \Drupal\data_policy\Entity\UserConsentInterface */
```

Comment:
I see that it's old code but will be good to make the following small fix here:
```suggestion
    /* @var \Drupal\data_policy\Entity\UserConsentInterface $entity */
```